test/Microsoft.AspNet.Mvc.TagHelpers.Test/LabelTagHelperTest.cs

```diff
@@ -44,45 +44,95 @@ public class LabelTagHelperTest
                 return new TheoryData<object, Type, Func<object>, string, TagHelperOutputContent>
                 {
                     { null, typeof(Model), () => null, "Text",
-                        new TagHelperOutputContent(Environment.NewLine, "Text", "Text") },
+                        new TagHelperOutputContent(string.Empty, Environment.NewLine, Environment.NewLine, "Text") },
+                    { null, typeof(Model), () => null, "Text",
+                        new TagHelperOutputContent(Environment.NewLine, string.Empty, "Text", "Text") },
 
                     { modelWithNull, typeof(Model), () => modelWithNull.Text, "Text",
-                        new TagHelperOutputContent(Environment.NewLine, "Text", "Text") },
+                        new TagHelperOutputContent(string.Empty, Environment.NewLine, Environment.NewLine, "Text") },
+                    { modelWithNull, typeof(Model), () => modelWithNull.Text, "Text",
+                        new TagHelperOutputContent(Environment.NewLine, string.Empty, "Text", "Text") },
+                    { modelWithText, typeof(Model), () => modelWithText.Text, "Text",
+                        new TagHelperOutputContent(string.Empty, Environment.NewLine, Environment.NewLine, "Text") },
+                    { modelWithText, typeof(Model), () => modelWithText.Text, "Text",
+                        new TagHelperOutputContent(Environment.NewLine, string.Empty, "Text", "Text") },
+                    { modelWithText, typeof(Model), () => modelWithNull.Text, "Text",
+                        new TagHelperOutputContent(string.Empty, "Hello World", "Hello World", "Text") },
+                    { modelWithText, typeof(Model), () => modelWithText.Text, "Text",
+                        new TagHelperOutputContent(string.Empty, "Hello World", "Hello World", "Text") },
+                    { modelWithText, typeof(Model), () => modelWithNull.Text, "Text",
+                        new TagHelperOutputContent("Hello World", null, null, "Text") },
                     { modelWithText, typeof(Model), () => modelWithText.Text, "Text",
-                        new TagHelperOutputContent(Environment.NewLine, "Text", "Text") },
+                        new TagHelperOutputContent("Hello World", null, null, "Text") },
                     { modelWithText, typeof(Model), () => modelWithNull.Text, "Text",
-                        new TagHelperOutputContent("Hello World", "Hello World", "Text") },
+                        new TagHelperOutputContent("Hello World1", "Hello World2", "Hello World2", "Text") },
                     { modelWithText, typeof(Model), () => modelWithText.Text, "Text",
-                        new TagHelperOutputContent("Hello World", "Hello World", "Text") },
+                        new TagHelperOutputContent("Hello World1", "Hello World2", "Hello World2", "Text") },
```

Comment:
It essentially didn't change, I added a new parameter to the start of `TagHelperOutputContent` to represent the existing `Output.Content` value. Therefore if I hadn't have done that it'd look like:

`new TagHelperOutputContent("Hello World", "Hello World", "Text")` =>
`new TagHelperOutputContent("Hello World2", "Hello World2", "Text")`

I essentially added 1s and 2s to indicate that we were in-fact getting our expected value.